src/streamlink/plugins/funimationnow.py

```diff
@@ -238,7 +238,7 @@ def _get_streams(self):
                 for subtitle in exp.subtitles():
                     log.debug(f"Subtitles: {subtitle['src']}")
                     if subtitle["src"].endswith(".vtt") or subtitle["src"].endswith(".srt"):
-                        sub_lang = {"en": "eng", "ja": "jpn"}[subtitle["language"]]
+                        sub_lang = {"en": "eng", "ja": "jpn", "es": "esp", "pt": "por"}[subtitle["language"]]
```

Comment:
Thanks for the PR, but could you please fix the key lookup as well, so that `KeyError`s won't get raised when there's a subtitle language that doesn't exist in the dictionary?

I'd suggest turning the `sub_lang` dictionary into a class member (either on the `Experience` class or the `FunimationNow` class) and changing the lookup to:

```suggestion
                        sub_lang = Experience.sub_langs.get(subtitle["language"])
                        if not sub_lang:
                            continue
```

Thanks